src/components/Calculator.js

```diff
@@ -0,0 +1,58 @@
+import React from 'react';
+import '../App.css';
+import PropTypes from 'prop-types';
+
+function CalcButton({ color, content }) {
```

Comment:
- [ ] Kindly use each file with its own component. For this CalcButtton component you have implemented here you can place it in another file in order to keep your file neat and clean because in later milestones there will be a lot you will have to add and you want everything to be as readable as possible.